dynawo/sources/Modeler/DataInterface/DYNReactiveCurvePointsInterface.h

```diff
@@ -34,11 +34,11 @@ class ReactiveCurvePointsInterface {
     /**
      * @brief Constructor
      *
-     * @param p active power
-     * @param qmin minimum reactive power
-     * @param qmax maximum reactive power
+     * @param p_ active power
+     * @param qmin_ minimum reactive power
+     * @param qmax_ maximum reactive power
      */
-    ReactiveCurvePoint(double p, double qmin, double qmax) : p(p), qmin(qmin), qmax(qmax) {}
+    ReactiveCurvePoint(double p_, double qmin_, double qmax_) : p(p_), qmin(qmin_), qmax(qmax_) {}
```

Comment:
coding style is wrong here, p_ are for attributes, you can correct it also in this pr

Reply:
I know but it will break public interface if I change struct attributes names !
I have choose to change parameters names, so...
If you want, I can change parameters names with a p_ prefix (p is for parameter) : p_p for p, p_qmin for qmin and p_qmax for qmax.
Are you agree ?

Reply:
ok i prefer it like this